templates/taxonomy-topic.php

```diff
@@ -1,10 +1,53 @@
 <?php
 get_header();
+
+$tags = array();
+$faqs = array();
 $container_classes = " container mb-5";
 $show = ( get_field( 'faq-topic-show-answers', get_queried_object() ) ) ?  " show" : "";
 $topic_description = term_description();
+
+
+function display_faq( $post, $show, $question_class ) {
```

Comment:
We should avoid defining functions like `display_faq()` and `get_related_posts()` in template files like this--maybe make them static methods off of the `UCF_FAQ_List_Common` class?